src/System.Private.CoreLib/shared/System/MemoryExtensions.cs

```diff
@@ -536,11 +536,41 @@ public static int IndexOfAny<T>(this Span<T> span, ReadOnlySpan<T> values)
         {
             if (typeof(T) == typeof(byte))
             {
-                return SpanHelpers.IndexOfAny(
-                    ref Unsafe.As<T, byte>(ref MemoryMarshal.GetReference(span)),
-                    span.Length,
-                    ref Unsafe.As<T, byte>(ref MemoryMarshal.GetReference(values)),
-                    values.Length);
+                ref var valueRef = ref Unsafe.As<T, byte>(ref MemoryMarshal.GetReference(values));
```

Comment:
Nit: ref var => ref byte... same for other occurrences.

Reply:
Changed